blocks/editable/test/index.js

```diff
@@ -10,6 +10,28 @@ import Editable from '../';
 import { diffAriaProps, pickAriaProps } from '../aria';
 
 describe( 'Editable', () => {
+	describe( 'Component', () => {
+		describe( '.getSettings', () => {
+			const value = [ 'Hi!' ];
+			const wrapper = shallow( <Editable value={ value } /> );
+			const settings = {
+				setting: 'hi',
+			};
+
+			test( 'should return expected settings', () => {
+				expect( wrapper.instance().getSettings( settings ) ).toEqual( {
+					setting: 'hi',
+					forced_root_block: false,
+				} );
+			} );
+
+			test( 'should be overriden', () => {
+				const mock = jest.fn().mockImplementation( ( setting ) => 'mocked' ); // eslint-disable-line no-unused-vars
```

Comment:
It is perfectly valid to omit `setting` in function definition instead of disabling Eslint check.

Reply:
You can also pass callback to `jest.fn` instead of using `mockImplementation`. See https://facebook.github.io/jest/docs/en/mock-functions.html#mock-implementations.

```js
const mock = jest.fn( () => 'mocked'  );
```

Reply:
Yup, makes sense.